src/app/(proper_react)/redesign/(authenticated)/user/welcome/EnterInfo.tsx

```diff
@@ -102,17 +146,41 @@ export const EnterInfo = (props: Props) => {
       displayValue: new Date(dateOfBirth).toLocaleDateString("en-US", {
         dateStyle: "medium",
       }),
-      errorMessage: l10n.getString(
-        "onboarding-enter-details-input-error-message"
-      ),
+      // TODO: Localize string
+      errorMessage: "You have to be at least 13 years old",
```

Comment:
Note: Need to sync with Content.

Reply:
Interesting... Doesn't FxA do COPPA/age checking on their end already? So theoretically you can't sign up for a Firefox Account and sign in if you're <13.
Disclaimer: I am not a lawyer. :shockedpikachu: